### PR TITLE
#783 initial commit

### PR DESCRIPTION
**Pull request by abidingotter**
equella/Equella#783

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    //jcenter(),
+    mavenCentral()
+}
+
+dependencies {
+    // This dependency is found on compile classpath of this component and consumers.
+    implementation 'com.google.guava:guava:27.0.1-jre'
+
+    // Use TestNG framework, also requires calling test.useTestNG() below
+    testImplementation 'org.testng:testng:6.14.3'
```
Comment (edalex-ian) [inline build.gradle]: hmmm, could we perhaps use jUnit or Spock?

Comment (abidingotter) [reply build.gradle]: This was just in the default "gradle init" setup.  However the Equella autotests are using testNG also anyway.

Comment (edalex-ian) [reply build.gradle]: Gotcha.

But yes, although oEQ uses TestNG, I feel newer stuff should probably go with what seems to more common jUnit. And then if people are keen, I'd go Spock - but I appreciate this is then different.

Comment (abidingotter) [reply build.gradle]: Have switched to JUnit in my working branch.

**Pull request merged**